products/migrations/0001_initial.py

```diff
@@ -1,5 +1,6 @@
-# Generated by Django 3.2.7 on 2021-09-15 07:44
+# Generated by Django 3.2.7 on 2021-09-20 08:45
```

Comment:
BTW, since we have already merged products feat, this should have been in a new migration file. Imagine a case when someone already have the tables in place and already ran the initial migration and for them, it would not run. So we need to add these changes in a new migration file instead of updating an old one please.